release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4811,4 +4811,38 @@ With this update, the Ingress Operator does not add the `clientca-configmap` fin
 [id="ocp-4-12-60-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.12.61
+[id="ocp-4-12-61"]
+=== RHSA-2024:4677 - {product-title} 4.12.61 bug fix and security update
+
+Issued: 2024-07-25
+
+{product-title} release 4.12.61, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4677[RHSA-2024:4677] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:4679[RHBA-2024:4679] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.61 --pullspecs
+----
+
+[id="ocp-4-12-61-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `mapi_instance_create_failed` alert metric did not fire when there was an error for the Accelerated Networking feature on Microsoft Azure clusters. This release adds the missing alert so that clusters with Accelerated Networking enabled generate alerts when required. (link:https://issues.redhat.com/browse/OCPBUGS-5235[*OCPBUGS-5235*])
+
+* Previously, the `wait-for-ceo` command that is used during the bootstrap operation to verify etcd rollout did not report errors for some failure modes. With this release, a code fix ensures that error messages for these failure modes get reported. (link:https://issues.redhat.com/browse/OCPBUGS-35501[*OCPBUGS-35501*])
+
+* Previously, the Helm Plugin index view did not display the same number of charts as the Helm CLI if the chart names varied. With this release, the Helm catalog now looks for `charts.openshift.io/name` and `charts.openshift.io/provider` so that all versions are grouped together in a single catalog title. (link:https://issues.redhat.com/browse/OCPBUGS-34933[*OCPBUGS-34933*])
```

Comment:
A noun after `charts.openshift.io/name` and `.../provider` could be helpful. However, I see this is a backport and already published in 4.16 so I'd leave it.